lua/entities/gmod_wire_expression2/core/extloader.lua

```diff
@@ -81,7 +81,7 @@ end
 local function e2_include_pass2(name, luaname, contents)
 	local preprocessedSource = e2_extpp_pass2(contents)
 
-	E2Lib.currentextension = string.StripExtension(string.gsub(name,"custom/",""))
+	E2Lib.currentextension = string.gsub(string.gsub(name,"custom/",""),"%.lua","")
```

Comment:
`string.StripExtension(string.GetFileFromFilename(name))`

Reply:
I suppose we can use `string.GetFileFromName`, but `string.StripExtension` uses a string match AND a string sub for some reason.

Reply:
The point is readability. I think two gsub is probably just as bad anyway. Best performance would probably be `string.match(string.match(name, "[^\\/]*$"), "[^%.]*")` but its completely unreadable.

Reply:
Alright, went back to your way.